src/main/java/mate/academy/model/ShoppingCart.java

```diff
@@ -0,0 +1,73 @@
+package mate.academy.model;
+
+import java.util.ArrayList;
+import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.MapsId;
+import javax.persistence.OneToMany;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "shopping_carts")
+public class ShoppingCart {
+    @Id
+    private Long id;
+    @OneToOne
```

Comment:
```suggestion
    @OneToOne(fetch = FetchType.LAZY)
```
Please fix and test that solution is working.